### DISCUSSION TITLE
您好

### DISCUSSION THREAD

**Discussion by newSF** [dyphire/mpv-config#134] [Q&A] [answered]
每次打开新的文件后，左上角都会弹出一个提示框，其中选择“是”的确认键为Enter键，这与全屏的快捷键存在冲突。尽管可以更改全屏快捷键，但Enter键偏偏也是我的使用习惯，且在脚本或按键绑定中并未找到修改此功能的相关设置，想请教一下具体应如何操作才能取消这个功能。![Image_17817847348827.png](https://github.com/user-attachments/assets/36e50ee4-7240-429d-90c9-1c8f32aa3169)

**Comment by dyphire** [comment, marked as answer]
如果不需要此脚本功能，可以在对应的脚本配置文件中禁用，将 enabled 值改为 no 即可：
https://github.com/dyphire/mpv-config/blob/426fd87a7250d696a03636f1faf4332718b0437c/script-opts/history_bookmark.conf#L1-L2

之后我会更改脚本内的强制键绑定，不再覆盖 Enter/Esc 键位

**Comment by dyphire** [comment]
在提交 5cb32cf 中优化该脚本功能并更改键位绑定

**Reply by newSF** [reply]
非常感谢
